Basic/Pod/ParallelCPU.pod

```diff
@@ -91,14 +91,14 @@ performing processing.
 
 B<Example>
 
- $a = sequence(20,4,3); # Small 3-D Array, size 20,4,3
+ $x = sequence(20,4,3); # Small 3-D Array, size 20,4,3
  
  # Setup auto-pthreading:
  set_autopthread_targ(2); # Target of 2 pthreads
  set_autopthread_size(0); # Zero so that the small PDLs in this example will be pthreaded
 
  # This will be split up into 2 pthreads
- $c = maximum($a);
+ $c = maximum($x);
 
 For the above example, the I<maximum> function has a signature of C<(a(n); [o]c())>, which means that the first
 dimension of $a (size 20) is a I<Core> dimension of the I<maximum> function. The other dimensions of $a (size 4,3)
```

Comment:
line 104 needs `$a`->`$x` twice to keep it in line with the code on line 94.  But the signature references piddle a(n), so that might be confusing.  This brings up a related point that I haven't looked in to yet: are the _signatures_ of functions affected by these changes?

Reply:
Made the change. The signatures aren't changed and all existing code should still work since `$a` et al have no special meaning in PDL-land, only in Perl-land.